datasets/plugins/__init__.py

```diff
@@ -0,0 +1,8 @@
+# isort: skip_file
+# flake8: noqa: F401
+from datasets.plugins.executors.metaflow_executor import MetaflowExecutor
+from datasets.plugins.batch.batch_dataset_plugin import BatchDatasetPlugin
```

Comment:
You probably also want to expose `InvalidOperationException` so clients can catch it without needing to be overly specific importing it. Potentially also moving it into a `datasets.exceptions` module potentially would be nicer, but not sure how that'd play with `plugins` so either way works for me so long as the client can easily do `except InvalidOperationException` I'm happy!